CONTRIBUTING.md

```diff
@@ -134,6 +135,40 @@ cd test/manual/k8s_crds
 ```
 This test currently does not run as part of the test suite.
 
+## Documentation
+Secretless has a few sources for documentation: a website, a documentation subdomain, and godocs.
+
+### Website
+The [website](https://secretless.io) source is in the [docs](docs/) folder in this repository. It is generated using Jekyll.
+
+#### Prerequisites
+
+To get the site up and running locally on your computer, ensure you have:
+1. Ruby version 2.1.0 or higher (check by running `ruby -v`)
+2. Bundler (`gem install bundler`)
+3. Jekyll (`gem install jekyll`)
+4. Once Bundler and Jekyll gems are installed, run `bundle install`
```

Comment:
Shouldn't you only have to run `bundle install` if you've added the deps to your Gemfile?  

But if we're locally running `gem install bundler` and `gem install jekyll` you shouldn't also need to run `bundle install`?

Reply:
I dunno - I just copied the instructions that were in the docs/README, and they've never let me down :)